### PR TITLE
[SYSTEMML-1110] Run data tests as suite

### PR DESCRIPTION
**Pull request by gweidner**
Switched functions.data tests containing FullReblockTest to run under Suite class to prevent them running at the same.

### CODE REVIEW

**Comment by gweidner** [comment]
All tests passed at:
https://sparktc.ibmcloud.com/jenkins/job/SystemML-OnDemand/246/

This change is intended to help potentially reduce intermittent failures but does not restrict any test outside the package running in parallel.

File: pom.xml [hunk 1]
```diff
@@ -386,6 +387,7 @@
 
 					<excludes>
 						<exclude>**/slowtest/**</exclude>
+						<include>**/integration/functions/data/*Test*.java</include>
```
Comment (deroneriksson) [inline pom.xml]: Hi @gweidner! L390 looks to have an ```<include>``` element inside the ```<excludes>``` element.

**Comment by gweidner** [comment]
Great catch Deron - thank you for reviewing!  I pushed commit to correct the exclude statement.
Tests in progress:
https://sparktc.ibmcloud.com/jenkins/job/SystemML-PullRequestBuilder/1041/
Previous successful PR build results here:
https://sparktc.ibmcloud.com/jenkins/job/SystemML-PullRequestBuilder/1040/

**Comment by deroneriksson** [comment]
LGTM. All tests passed: https://sparktc.ibmcloud.com/jenkins/job/SystemML-PullRequestBuilder/1041/

**Pull request closed without merging**